openbadges/.htaccess

```diff
@@ -1,4 +1,5 @@
 Options +FollowSymLinks
 RewriteEngine on
 RewriteRule ^v1$ https://dev.truecred.com/contexts/openbadges-v1.jsonld [R=302,L]
+RewriteRule ^v1.1$ http://specification.openbadges.org/1.1/context.json [R=302,L]
```

Comment:
I'm pretty wary about using version identifiers in namespaces, certainly once a spec is stable, the namespace URI should never change. That's why we're still stuck with `http://xmlns.com/foaf/0.1/` after all these years, it could never be changed.

I would say that once the w3id.org URI is determined, that is what distinguishes it from a previous version such as truecred or some other interim identifier.